tools/data_exploration/anonymization.xml

```diff
@@ -0,0 +1,69 @@
+<tool id="tool_anonymization" name="Spatial coordinates anonymization" version="0.0" profile="20.01">
```

Comment:
ID should be adopted to fit the others, version maybe 0.0.1?

Reply:
Indeed, I will change the version, but for the id I am not sure what to write because this tool can be use on any coordinates data and not just on ecology data